From 42381683097e61113f8cf4b13795a2e03e566835 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Sun, 19 Dec 2021 04:45:25 +0800
Subject: [PATCH] Fix iniparser.h include

In configure.ac `iniparser.h` was checked without a path too, this change makes it consistent and fixes building on Arch Linux which has the header installed in /usr/include directly.
---
 util/parse-configs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/parse-configs.c b/util/parse-configs.c
index 1b7ffa6..c834a07 100644
--- a/util/parse-configs.c
+++ b/util/parse-configs.c
@@ -4,7 +4,7 @@
 #include <dirent.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <iniparser/iniparser.h>
+#include <iniparser.h>
 #include <sys/stat.h>
 #include <util/parse-configs.h>
 #include <util/strbuf.h>
-- 
2.30.2

